src/nnet3bin/nnet3-copy.cc

```diff
@@ -0,0 +1,76 @@
+// nnet3bin/nnet3-copy.cc
+
+// Copyright 2012  Johns Hopkins University (author:  Daniel Povey)
```

Comment:
Please add something here like
//                            2015   Xingyu Na
after the first copyright line, to reflect that you worked on it.  (I imagine you are a student; this is normally what students should do if they are not doing this as part of some kind of employment).